source/2019-12-20-ember-3-15-released.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Ember 3.15 "Octane" Released
+author: Kenneth Larsen, Matthew Beale, Melanie Sumner
+tags: Releases, 2019, 3, 3.15, Version 3.x
+alias: 2019-12-20-ember-3-15-released.md
+responsive: true
+---
+
+Today the Ember project is releasing version 3.15 of Ember.js, Ember Data, and
+Ember CLI. Ember 3.15 has been dubbed "Octane" as Ember's first [edition release](https://emberjs.com/editions/).  
+
+Ember developers using the Octane edition can exercise modern features like native ES
+classes, decorators, the new Glimmer component API, and a new approach to state
+management called tracked properties. Combined, these innovative changes to
+<!--alex ignore just-->
+Ember are more than just new tools and APIs. Octane's features, together,
+sum up to a paradigm shift in how Ember applications are built.
+
+READMORE
+
+<!--alex ignore mental-->
+Read more about what we call the "mental model" of working with Octane in the
+upcoming Octane announcement blog post from Yehuda.
+
+...and then explore our completely updated and refreshed guide and API
+documentation:
+
+* Check out the completely rewritten [core concepts](https://guides.emberjs.com/release/components/) in the guides.
+* Read the [completely updated Super Rentals tutorial](https://guides.emberjs.com/release/tutorial/part-1/).
+* Already using Ember and want to adopt Octane? See the [Octane Upgrade
+  Guide](/release/upgrading/current-edition/) for the most complete
```

Comment:
I think this should be an absolutely path too (since the blog is at `blog.emberjs.com` and the content will be at `guides.emberjs.com`)
```suggestion
  Guide](https://guides.emberjs.com/release/upgrading/current-edition/) for the most complete
```